schematic/store/synapse.py

```diff
@@ -538,6 +538,88 @@ def getDataTypeFromManifest(self, manifestId:str):
 
         return result_dict
 
+    def _get_files_under_datasets(self, datasetId: str) -> dict:
```

Comment:
change it to `_get_files_metadata_from_dataset` since we are not actually getting the files